controllers/webscrapeController.js

```diff
@@ -26,6 +26,7 @@ async function scrapeEspn(startDate, endDate, res) {
 
     const current = new Date(start);
 
+    // eslint-disable-next-line no-unmodified-loop-condition
```

Comment:
look into adding eslint config file to disable specific cases instead of commenting like this